docs/content/preview/yugabyte-voyager/migrate/live-fall-forward.md

```diff
@@ -51,28 +51,512 @@ Before proceeding with migration, ensure that you have completed the following s
+      LOOP
+        EXECUTE 'ALTER TABLE ' || cur_table || ' OWNER TO replication_group';
+      END LOOP;
+    END $$;
+    ```
+
+1. Grant `CREATE` privilege on the source database to `ybvoyager` as follows:
+
+    ```sql
+    GRANT CREATE ON DATABASE <database_name> TO ybvoyager; --required to create a publication.
+    ```
+
+  {{% /tab %}}
+
+  {{% tab header="RDS PostgreSQL" %}}
+
+1. yb_voyager requires `wal_level` to be logical. This is controlled by a database parameter `rds.logical_replication` which needs to be set to 1. You can check this using following the steps:
+
+    1. Run the command `SHOW rds.logical_replication` on the database to check whether the parameter is set.
+
+    1. If the parameter is not set, you can change the parameter value to 1 from the RDS console of the database; navigate to **Configuration** > **Parameter group** > `rds.logical_replication`.
+
+    1. If the `rds.logical_replication` errors out (after the change), create a new parameter group with the value as 1, and assign it to the database instance from the **Modify** option on the RDS console.
+
+    1. Restart RDS.
+
+1. Check that the replica identity is FULL (`f`) for all tables on the database.
+
+    1. Check the Replica identity for all the tables on the database as follows:
+
+        ```sql
+        SELECT relname, relreplident
+        FROM pg_class
+        WHERE relnamespace = (SELECT oid FROM pg_namespace WHERE nspname = '<source_schema_name>') AND relkind = 'r';
+        ```
+
+    1. Change the replica identity of all tables if the tables have an identity other than FULL (`f`), using the following query:
+
+        ```sql
+        DO $$
+        DECLARE
+          table_name_var text;
+        BEGIN
+          FOR table_name_var IN (SELECT table_name FROM information_schema.tables WHERE table_schema = '<source_schema_name>' AND table_type = 'BASE TABLE')
+          LOOP
+            EXECUTE 'ALTER TABLE ' || table_name_var || ' REPLICA IDENTITY FULL';
+          END LOOP;
+        END $$;
+        ```
+
+1. Create user `ybvoyager` for the migration using the following command:
+
+    ```sql
+    CREATE USER ybvoyager PASSWORD 'password';
+    GRANT rds_replication to ybvoyager;
+    ```
+
+1. Switch to the database that you want to migrate as follows:
+
+   ```sql
+   \c <database_name>
+   ```
+
+1. Grant the `USAGE` permission to the `ybvoyager` user on all schemas of the database as follows:
+
+   ```sql
+   SELECT 'GRANT USAGE ON SCHEMA ' || schema_name || ' TO ybvoyager;' FROM information_schema.schemata; \gexec
+   ```
+
+   The above `SELECT` statement generates a list of `GRANT USAGE` statements which are then executed by `psql` because of the `\gexec` switch. The `\gexec` switch works for PostgreSQL v9.6 and later. For older versions, you'll have to manually execute the `GRANT USAGE ON SCHEMA schema_name TO ybvoyager` statement, for each schema in the source PostgreSQL database.
```

Comment:
```suggestion
   The preceding `SELECT` statement generates a list of `GRANT USAGE` statements which are then executed by `psql` because of the `\gexec` switch. The `\gexec` switch works for PostgreSQL v9.6 and later. For older versions, you'll have to manually execute the `GRANT USAGE ON SCHEMA schema_name TO ybvoyager` statement, for each schema in the source PostgreSQL database.
```